setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
```

Comment:
I'd like to follow semver, and I'm not too keen on committing to a stable API at this point. `0.1` would work well, I thinkI'd like to follow semver, and I'm not too keen on committing to a stable API at this point. `0.1` would work well, I think